bin/backup-all-my-flickr-photos

```diff
@@ -17,7 +17,7 @@ import socket
 import errno
 
 _forbidden_characters = r'<>:"/\|?*'
-_extensions = ["jpg", "mov", "mp4", "png", "flv", "gif"]
+_extensions = ["jpg", "mov", "mp4", "png", "flv", "gif", "TBD"]
```

Comment:
`.tbd` is a file extension that is used by some programs. I would prefer using a file extension that is clearly unique, maybe `.unknown`. Also, I would prefer filename extensions to be in lower-case, to make life simpler on operating systems with case-sensitive filesystems.

Reply:
Works for me as long as this can ingest whatever Flickr sends down the pipe.